src/modules/app/helpers/parse-query.js

```diff
@@ -0,0 +1,21 @@
+/**
+  * Parses the raw query string into a formatted array
+  * @param {string} searchString - The raw query param string
+  * @returns {object}
+*/
+export default function parseSearch(searchString) {
```

Comment:
Should probably rename this function to `parseQuery` to match its name when imported (and to match the name of the file).